nupic/embodied/envs/real_robots/policies.py

```diff
@@ -19,14 +19,15 @@
 #  http://numenta.org/licenses/
 #
 # ------------------------------------------------------------------------------
-from real_robots.policy import BasePolicy
 import numpy as np
+from real_robots.policy import BasePolicy
```

Comment:
Ordering of sorts, the standards come from PEP8: https://www.python.org/dev/peps/pep-0008/#imports